docs/gestores.md

```diff
@@ -0,0 +1,46 @@
+# Instalación
+He decidido usar poetry pues fue el primer gestor del que oí hablar, busqué información sobre este y vi lo sencillo que era sobre Invoke y la documentación que aportaba en su web me ayudo a entender el uso de los test y control de dependencias
```

Comment:
No se pide que comuniquéis una decisión, sino que establezcáis los parámetros para tomarla y la justifiquéis.